README.md

```diff
@@ -1887,6 +1892,38 @@ Parameters:
 
 `mount_path:` Defines the directory where the device will be mounted (e.g., /dir/to/persistence).
 
+## Client count scaling
+By default, each simulator worker uses a single Hazelcast instance. For workers using the Hazelcast Java client the number of 
+clients is configurable and can be increased. This can be useful when benchmarking how a Hazelcast cluster behaves with a high 
+number of client connections as having a single client per worker process is expensive with thousands of clients. To configure this
+value you can set the `clients_per_loadgenerator` value in the `tests.yaml` file. For example the following snippet will configure
+600 loadgenerator processes each with 10 Hazelcast client instances
+
+```yaml
+- name: many_clients_test
+  duration: 5m
+  repetitions: 1
+  node_count: 10
+  loadgenerator_count: 600
+  clients_per_loadgenerator: 10
+  driver: hazelcast-enterprise5
+  # ...
+```
+
+How the clients are used is **test specific** and most current test implementations simply ignore all but one of the clients. One
+test which works with multiple clients is `LongByteArrayMapTest` which evenly assigns simulator threads to clients in a many to 1 
+fashion. For example the following config will assign each simulator test thread a single client.
+
+```yaml
+    - class: com.hazelcast.simulator.tests.map.LongByteArrayMapTest
+      name: map
+      threadCount: 10
+      getProb: 0.9
+      putProb: 0.1
+```
+
+With `threadCount: 20` each client would have 2 threads assigned to it. If `threadCount < clients_per_loadgenerator` then the 
+extra clients will simply be ignored.
```

Comment:
can you also add a code snippet to show how in new tests people can use these instances?